datatree/tests/test_datatree.py

```diff
@@ -440,6 +440,38 @@ def test_setitem_dataarray_replace_existing_node(self):
         xrt.assert_identical(results.to_dataset(), expected)
 
 
+class TestMoveNode:
+    dt = DataTree.from_dict(
+        d={
+            "/": xr.Dataset({"name": "Hobert"}),
+            "/Jake": xr.Dataset({"name": "Jake"}),
+            "/Bragg": xr.Dataset({"name": "Bragg"}),
+            "/Bragg/Kyle": xr.Dataset({"name": "Noah"}),
+            "/Kai": xr.Dataset({"name": "Kai"}),
+        },
+        name="TestMove",
+    )
+
+    def test_move_to_root(self):
```

Comment:
This test looks useful, but it doesn't actually move anything to the root position like the name suggests.